playbooks/SOC Simulator/Create Phishing Alert.json

```diff
@@ -0,0 +1,218 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "Create Phishing Alert",
```

Comment:
Why we need this PB?

Reply:
Hi @cs-suyog-jinturkar 

I have removed this playbooks also made below changes:

- Removed extra playbooks from the playbook collection
   > Create Alert
   > Create Phishing Alert
   > Create Zero day Malicious File Indicator
   > Get Malicious IP and URL
   > Replace Variables Values

- Updated Playbook collection name to "02 - Use Case - SOC Simulator"

- Updated info.json accordingly